charts/rancher-cis-benchmark/0.1.2/questions.yaml

```diff
@@ -1 +1,2 @@
 rancher_min_version: 2.4.6-rc1
+rancher_max_version: 2.5.8
```

Comment:
Hmm. We could make this 2.5.99 for when a new version gets introduced in 2.5.x, but I suppose this works for now since its in 2.6 branch and can't be affected. We need some max so that it doesn't get included in images.txt.